Modules/mathmodule.c

```diff
@@ -3001,8 +3001,8 @@ math_prod_impl(PyObject *module, PyObject *iterable, PyObject *start)
 /*[clinic input]
 math.comb
 
-    n: object(subclass_of='&PyLong_Type')
-    k: object(subclass_of='&PyLong_Type')
+    n: object
+    k: object
```

Comment:
Should these be positional-only? n and k does not sound very descriptive to me and the call can be faster